src/main/java/org/broadinstitute/hellbender/tools/exome/cnlohcaller/CNLOHCaller.java

```diff
@@ -0,0 +1,803 @@
+        }
+
+        if (val >= credibleMode) {
+            return new NormalDistribution(credibleMode, hiDist / 1.96).density(val);
+        } else {
+            return new NormalDistribution(credibleMode, loDist / 1.96).density(val);
+        }
+    }
+
+    /**
+     * Calculate a copy ratio.
+     *
+     * @param rho CCF * purity
+     * @param m Absolute copy number of allele 1
+     * @param n Absolute copy number of allele 2
+     * @param lambda CCF * ploidy
+     * @return copy ratio in CR space
+     */
+    @VisibleForTesting
+    static double calculateCopyRatio(final double rho, final int m, final int n, final double lambda) {
+        return ((1-rho) * 2 + rho*(n+m)) / lambda;
+    }
+
+    /**
+     * Calculate the likelihood for the copy ratio.  Models the credible interval as a Gaussian.
+     *
+     * @param rho -- see class docs
+     * @param m -- see class docs
+     * @param n -- see class docs
+     * @param lambda -- see class docs
+     * @param credibleMode mode of the credible interval from for the copy ratio.
+     * @param credibleLow  low point of the credible interval from for the copy ratio.
+     * @param credibleHigh  high point of the credible interval from for the copy ratio.
+     * @param additionalVariance Additional variance that should be included in the segment mean pdf
+     * @return likelihood, with a minimum value of {@link CNLOHCaller#MIN_L}
+     */
+    @VisibleForTesting
+    static double calculateFcr(final double rho, final int m, final int n, final double lambda, final double credibleMode,
+                         final double credibleLow, final double credibleHigh, final double additionalVariance) {
+        ParamUtils.inRange(rho, 0.0, 1.0, "Invalid rho value: " + rho + ".  Must be [0,1]");
+        ParamUtils.isPositiveOrZero(m, "M must be positive.");
+        ParamUtils.isPositiveOrZero(n, "N must be positive.");
+        ParamUtils.isPositiveOrZero(additionalVariance, "additional variance must be positive.");
+        if (Double.isNaN(credibleMode) || Double.isNaN(credibleLow) || Double.isNaN(credibleHigh)){
+            return 1;
+        }
+        final double cr = calculateCopyRatio(rho, m, n, lambda);
+        final double avgDist = ((credibleMode - credibleLow) + (credibleHigh - credibleMode))/2.0;
+        return Math.max(new NormalDistribution(credibleMode, (avgDist / 1.96) + Math.sqrt(additionalVariance)).density(cr), MIN_L);
+    }
+
+    /**
+     * Attempt to get an idea of segment mean variance near copy neutral.
+     *
+     * @param segments Never {@code null}
+     * @return variance of segment mean (in CR space) of segments that are "close enough" to copy neutral.
+     *   Zero if no segments are "close enough"
+     */
+    private double calculateVarianceOfCopyNeutralSegmentMeans(final List<ACNVModeledSegment> segments, final double meanBiasInCR) {
+        Utils.nonNull(segments);
+
+        final Variance varianceCalculator = new Variance();
```

Comment:
Rather than partitioning this as 1) construct calculator; 2) evaluate a big 'ol thing, I would do
```java
final double neutralCR = 1 + meanBiasInCR;
final double[] neutralSegmentMeans = segments.stream()
    .mapToDouble(ACNVModeledSegment::getSegmentMeanInCRSpace)
    .filter(m -> Math.abs(m - neutralCR) < 0.1).toArray()
return new Variance().evaluate(neutralSegmentMeans);
```